build.sbt

```diff
@@ -199,6 +199,10 @@ dependencyOverrides += guava
 sharedSettings
 
 libraryDependencies ++= playDeps
+
+resolvers ++= Seq(Resolver.mavenLocal) //, "Sonatype snapshots repository" at "https://oss.sonatype.org/content/repositories/snapshots/")
```

Comment:
need to ask pac4j guys to publish a new clean snapshot of `pac4j*` with Kerberos